src/features/wisdom/treeView.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
tree view is independent of wisdom service, can we move this file to top level under `src`?